homeshick.sh

```diff
@@ -1,12 +1,15 @@
 # This script should be sourced in the context of your shell like so:
-# source $HOME/.homeshick/repos/.homeshick/homeshick.sh
+# source homeshick.sh
 # Once the homeshick() function is defined, you can type
 # "homeshick cd CASTLE" to enter a castle.
 
-function homeshick() {
-	if [ "$1" = "cd" ] && [ -n "$2" ]; then
-		cd "$HOME/.homesick/repos/$2"
+__DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+eval "function homeshick() {
```

Comment:
I was wondering. Whatb is the motivation behind wrapping this in an eval, what problem does it solve?